DashAI/back/models/scikit_learn/bow_text_classification_model.py

```diff
@@ -22,7 +22,7 @@ class BagOfWordsTextClassificationModelSchema(BaseSchema):
 
     tabular_classifier: schema_field(
         component_field(parent="TabularClassificationModel"),
```

Comment:
In the future, do you think it will be possible to receive classes directly instead of their names?